liualgotrader/models/portfolio.py

```diff
@@ -191,3 +192,21 @@ async def get_external_account_id(
             )
 
             return (r["external_account_id"], r["broker"])
+
+    @classmethod
+    async def list_portfolios(cls) -> List[str]:
+        try:
+            pool = config.db_conn_pool
+        except AttributeError:
+            await create_db_connection()
+            pool = config.db_conn_pool
+
+        async with pool.acquire() as con:
+            async with con.transaction():
+                rows = await con.fetch(
+                    """
+                        SELECT DISTINCT portfolio_id
```

Comment:
I suggest returning the full details:

SELECT
portfolio_id , size,  parameters ,assets,  tstamp, broker, external_account_id
FROM
portfolio
WHERE 
expire_tstamp is not null;

you can use the `tabulate` package which is already part of the project to display it nicely (and explain the columns?)

I hope that makes sense. Thanks for handling!